pkg/k8shandler/cluster.go

```diff
@@ -274,3 +306,89 @@ func (cState *ClusterState) amendPods(dpl *v1alpha1.Elasticsearch) error {
 	}
 	return nil
 }
+
+func updateClusterSettings(dpl *v1alpha1.Elasticsearch) error {
+
+	event, err := getClusterEvent(dpl)
+	if err != nil {
+		return err
+	}
+	fmt.Printf("Event is: %s\n", event)
+
+	switch event {
+	case v1alpha1.ScaledDown:
+		if err := execClusterSettingsUpdate(dpl); err != nil {
+			return err
+		}
+		utils.UpdateClusterSettingsUpdateNeededCondition(&dpl.Status, v1alpha1.ConditionFalse)
+		utils.UpdateScalingDownCondition(&dpl.Status, v1alpha1.ConditionFalse)
+		break
```

Comment:
I don't think we actually need to do an explicit `break` for `switch` statements in go